ci_framework/roles/use_install_yamls/defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_use_install_yamls"
+cifmw_use_install_yamls_makefile: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/Makefile"
```

Comment:
in fact... that one is useless, since you point to the repository itself at the end of the file. We can then use
`{{ cifmw_use_install_yamls_repo }}/Makefile` instead.